Sources/LiveKit/Broadcast/BroadcastScreenCapturer.swift

```diff
@@ -53,22 +48,32 @@ class BroadcastScreenCapturer: BufferCapturer {
             .toEncodeSafeDimensions()
 
         set(dimensions: targetDimensions)
+        return createReceiver()
```

Comment:
might be worth adding guards somewhere that there isn't already a receiver set up